src/App.jsx

```diff
@@ -91,6 +102,21 @@ const App = () => {
           <Route path="/about" element={<LandingMain />}></Route>
           <Route path="/contact" element={<LandingMain />}></Route>
           <Route path="/" element={<LandingMain />}></Route>
+          <Route path="/currencyconverter" 
```

Comment:
the Transfer component shall be refactored in order to support that functionality